apps/ais/src/environments/version.ts

```diff
@@ -1,5 +1,5 @@
 export const version = {
   "semver": {
-    "version": ""
+    "version": "2.4.0"
```

Comment:
Hmmm... wo kommen diese Einträge her? Wurden die von Hand gemacht oder irgendwie durch GitHub/deploy/build?

Das ist noch irgendwie eine offene Baustelle, siehe auch #750 (aber nicht in diesem PR!)